README.md

```diff
@@ -61,6 +62,38 @@ The SDKs provide the following functionality:
 - Access token can be automatically and continuously refreshed
 - Redirect callback is invoked after login or register
 
+## Automated testing
+The [Playwright](https://playwright.dev/docs/intro) end-to-end tests verify the proper functionality of several authentication and authorization endpoints in the FA SDK.
+
+Prerequisites
+- Ensure Playwright has been installed as a dependency 
+- On a seperate server run the FA SDK consuming quickstart application.
+
+Configuration
+The Playwright configuration (playwright.config.ts) includes settings for running tests in parallel, specifying browser environments, and defining the base URL the tests will perform on.
+
+Running the Tests
+Prior to running the tests you will want to check the Server Command that is used to start up a local instance of the FA SDK consuming quickstart application. Additionally you will want to note the port number this application will run on.
+
+Run Tests:
+`SERVER_COMMAND="your-server-start-command" PORT=your-port-number yarn test:e2e`
+    Example: `SERVER_COMMAND="npm run start" PORT=9011 yarn test:e2e`
+
+The tests cover the following endpoints and functionalities:
```

Comment:
If it were up to me, I'd remove this line and below in favor of keeping this doc from getting bloated. It seems like TMI. That's my vote, but it's up to you @BriDavidson since you're the owner of the E2Es and the docs for them.

Reply:
I was thinking this as well but wasn't sure how detailed we wanted to get with this. I removed this.